src/core/store/modules/labs/labs.selectors.ts

```diff
@@ -165,6 +165,9 @@ const selectSummaryData = createSelector([selectDepositedLabs], (depositedLabs)
   depositedLabs.forEach((lab) => (totalDeposited = totalDeposited.plus(lab[lab.mainPositionKey].userDepositedUsdc)));
 
   return {
+    apy: calculateCombinedApy(
+      depositedLabs.map(({ apyMetadata, DEPOSIT: { userDepositedUsdc } }) => [apyMetadata?.net_apy, userDepositedUsdc])
+    ),
     totalDeposits: totalDeposited.toString(),
     totalEarnings: '0',
     estYearlyYeild: '0',
```

Comment:
Lets populate these ones, based on the additional context of the issue https://github.com/yearn/yearn-finance-v3/issues/531#issuecomment-1081321273

Reply:
Done with https://github.com/yearn/yearn-finance-v3/pull/562/commits/35b7318b06ad491e43fe2b9b8295a38c7f7adb15